easybuild/easyconfigs/l/libevent/libevent-2.1.8.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.1.8'
+
+homepage = 'http://libevent.org/'
+description = """The libevent API provides a mechanism to execute a callback function when a specific
+ event occurs on a file descriptor or after a timeout has been reached.
+ Furthermore, libevent also support callbacks due to signals or regular timeouts."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [
+    'https://github.com/downloads/%(name)s/%(name)s/',
+    'https://sourceforge.net/projects/levent/files/%(name)s/%(name)s-%(version_major_minor)s/',
```

Comment:
@foxik0070 auto-downloading the sources doesn't seem to work with these `source_urls`

We recently fixed the `source_urls` for `libevent`, cfr. #4037

So, please use the ones we're using now, see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/l/libevent/libevent-2.0.22-GNU-4.9.3-2.25.eb#L13